src/Dialog/Spinner.jsx

```diff
@@ -35,7 +35,7 @@
  */
 
 import React from 'react';
-import spinnerImg from '../../resources/ajax-loader.gif';
+import spinnerImg from './ajax-loader.gif';
```

Comment:
Funny old bugger this spinner is, it would be nice to replace it with a more elegant one sometime.